installers-conda/resources/post-install.bat

```diff
@@ -1,17 +1,36 @@
-rem This script launches Spyder after install
+@rem  This script launches Spyder after install
 @echo off
 
+call :redirect 2>&1 >> %PREFIX%\install.log
+
+:redirect
+@echo Environment Variables:
+set
+
+if defined CI set no_launch_spyder=true
+if "%INSTALLER_UNATTENDED%"=="1" set no_launch_spyder=true
+@echo CI = %CI%
+@echo INSTALLER_UNATTENDED = %INSTALLER_UNATTENDED%
+@echo no_launch_spyder = %no_launch_spyder%
+if defined no_launch_spyder (
+    @echo Installing in CI or silent mode, do not launch Spyder
```

Comment:
```suggestion
    @echo Installing in silent mode, do not launch Spyder
```

`CI` is technical term that perhaps users that install Spyder in silent mode don't know, so I think it's better not to mention it.